DuckDuckGo/AppTPFAQView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  AppTPFAQView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+#if APP_TRACKING_PROTECTION
+
+struct AppTPFAQView: View {
+    var faqBody: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Const.Size.stackSpacing) {
+                ForEach(AppTPFAQViewModel.faqs, id: \.question) { faq in
+                    VStack(alignment: .leading, spacing: Const.Size.stackSpacing) {
+                        Text(faq.question)
+                            .font(Font(uiFont: Const.Font.titleFont))
+                            .foregroundColor(Color.fontColor)
+                        Text(faq.answer)
+                            .font(Font(uiFont: Const.Font.contentFont))
+                            .foregroundColor(Color.fontColor)
+                    }
+                    .padding(Const.Size.stackPadding)
+                }
+            }
+            .frame(maxWidth: .infinity)
+        }
+        .navigationTitle(Text(UserText.appTPFAQTitle))
+        .navigationBarTitleDisplayMode(.inline)
+    }
+    
+    @ViewBuilder
+    var scrollWithBackgroud: some View {
+        if #available(iOS 16, *) {
+            faqBody
+                .scrollContentBackground(.hidden)
+                .background(Color(designSystemColor: .background))
+        } else {
+            faqBody
+                .background(Color(designSystemColor: .background))
+        }
+    }
+    
+    var body: some View {
+        scrollWithBackgroud
+    }
+}
+
+private enum Const {
+    enum Font {
+        static let titleFont = UIFont.boldAppFont(ofSize: 20)
```

Comment:
Fonts should all come from the Design System now.

Reply:
In figma you can see what font it should be by clicking into the text and it'll tell the Typography name.  Then in `DesignResourcesKit` there should be a corresponding font already defined.  If figma contains something not in the design system please push back on to the designer. 

<img width="238" alt="Screenshot 2023-06-30 at 13 37 44" src="https://github.com/duckduckgo/iOS/assets/81197/dd4019e6-84b8-4fa4-aaef-4198ebaa66f2">